test/common_nn.py

```diff
@@ -245,6 +245,10 @@ def get_weight(m):
     ),
 ]
 
+def poissonnllloss_reference(input, target, log_input=True, full=False, 
```

Comment:
Because PoissonNLLLoss is implemented in python, it doesn't make sense to have a reference function for it -- good catch and my apologies! Please remove this